2996_reflection/reflection.md

```diff
@@ -4907,37 +4904,9 @@ void f() {
 
 :::
 
-### [dcl.spec.auto.general]{.sref} Placeholder type specifiers {-}
-
-Extend the wording in [dcl.spec.auto.general]{.sref}/13 to account for splicing:
-
-::: std
-[13]{.pnum} If a variable or function with an undeduced placeholder type is [either]{.addu} named by an expression ([basic.def.odr]) [or designated by a `$splice-specifier$` in an expression]{.addu}, the program is ill-formed. Once a non-discarded `return` statement has been seen in a function, however, the return type deduced from that statement can be used in the rest of the function, including in other `return` statements.
```

Comment:
We don't need this anymore because we changed [basic.def.odr] to already account for this?